.cypress/fixtures/single_detector_index_response.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
should this be renamed to something like `single_detector_response` rather than saying `index` here? seems confusing to mention `index`

Reply:
sure. renamed it to `single_detector_response`